### PR TITLE
Add IT with new orchestrator

### PR DESCRIPTION
**Pull request by dbmeneses**
@tom-vanbraband-sonarsource Test with the new orchestrator. The orchestrator hasn't been released so this can't be merged

### CODE REVIEW

**Comment by dbmeneses** [comment]
@tom-vanbraband-sonarsource this has been updated to use the release orchestrator

File: it/src/test/java/com/sonarsource/scanner/it/SonarScannerTestSuite.java [hunk 1]
```diff
@@ -32,17 +34,17 @@
...
+      .getProperty("sonar.runtimeVersion", "LATEST_RELEASE[9.9]");
+    return OrchestratorRule.builderEnv()
       .useDefaultAdminCredentialsForBuilds(true)
-      .setSonarVersion(
-        sonarVersion).addPlugin(MavenLocation
-        .of("org.sonarsource.sonarqube", "sonar-xoo-plugin",
-          sonarVersion)).build();
+      .setSonarVersion(sonarVersion)
+      .addBundledPluginToKeep("sonar-javascript")
+      .addPlugin(MavenLocation.of("org.sonarsource.sonarqube", "sonar-xoo-plugin", sonarVersion))
```
Comment (tom-vanbraband-sonarsource) [inline it/src/test/java/com/sonarsource/scanner/it/SonarScannerTestSuite.java:46]: Would it make sense to create one orchestrator that only includes `sonar-javascript` and another that only includes `sonar-xoo-plugin`? Might be over-optimization, but most tests don't need the js plugin, and your new test does not need the xoo plugin

**Review by tom-vanbraband-sonarsource — APPROVED** [review]
Looks good! Validated that the time increase of the tests is minimal

File: it/src/test/java/com/sonarsource/scanner/it/SonarScannerTestSuite.java (hunk 1 above)
Comment (dbmeneses) [reply it/src/test/java/com/sonarsource/scanner/it/SonarScannerTestSuite.java:46]: given the number of tests, I believe starting a second instance would take longer than the small overhead introduced by the extra plugin.

**Pull request merged**